src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -61,6 +91,138 @@ class EmojiPickerMenu extends Component {
         if (this.props.forwardedRef && _.isFunction(this.props.forwardedRef)) {
             this.props.forwardedRef(this.searchInput);
         }
+
+        // Setup and attach keypress/mouse handlers only if we have a keyboard (and not a touchscreen)
+        // NOTE: these event handlers are instance members so we can reference
+        // the same functions in memory when removing them.
+        if (!canUseTouchScreen() && document) {
```

Comment:
I tested this on Mobile Web, and it turns out that Mobile Web also uses `index.native.js` instead of this file. Since this file will only be referenced by Desktop and Web (which should never have touchscreens and should always have a `document` object available), do you think it's still necessary to have this check?

@pecanoro @stitesExpensify

Reply:
hmm how did you come to that conclusion? I'm 90% sure that mobile web uses index.js

Reply:
I tried it on Safari in the simulator, and the search input doesn't appear (implying we're using `index.native.js`).

Reply:
Ahh wait, when I tried it on Safari in the simulator there was an error causing the search input not to appear leading me to think we were using `index.native.js`. Looks like it's still there and I need to include this check 👍

Reply:
This is actually what prevents the search bar 😄 https://github.com/Expensify/Expensify.cash/pull/2532/files#diff-0aeb1e7eb3f2813b84c81ee9aaf16b4e67307297dbc129803f5f0eb20a6a3693R313

In index.js we have a `isSmallScreenWidth`